README.md

```diff
@@ -5,17 +5,22 @@
 Table of Contents
 </summary>
 
-* [Getting Involved](#getting-involved)
-* [Governing Bodies](#governing-bodies)
-* [Communication](#communication)
-  * [Instant Messaging](#instant-messaging)
-  * [Calendar](#calendar)
-  * [Mailing Lists](#mailing-lists)
-  * [Community Meetings](#community-meetings)
-* [License](#license)
-* [Logos and Branding Guide](#logo-and-branding-guide)
-* [Special Interest Groups](#special-interest-groups)
-* [Related Groups](#related-groups)
+- [<img src="https://opentelemetry.io/img/logos/opentelemetry-logo-nav.png" alt="OpenTelemetry Icon" width="45" height=""> OpenTelemetry community content](#-opentelemetry-community-content)
```

Comment:
I assume this line is generated but looks pretty awkward 😄 
how about reverting the `Table of Contents` changes?

Reply:
![image](https://user-images.githubusercontent.com/5067549/124169944-1e704580-daa7-11eb-84ac-265e52e4ff5b.png)

Reply:
Sure can do that. Original `Table of Contents is anyway outdated, I can remove the `<img src=>` if that's the concern?

Reply:
Yeah. I would be OK with almost any option to get rid of e.g.;
1. Just remove the `<img>` from the Table of Content
2. Remove the whole line from the Table of Content
3. Remove the `<img` from the title header

Reply:
Thanks, have changed as per option 2 above.